august-19.md

```diff
@@ -1,62 +1,151 @@
+* In TiKV 3.0.2 we fixed the problem that TiKV lose some logs while panicking [#5174](https://github.com/tikv/tikv/pull/5174).
+* [@fullstop000] has a draft of Follower Replication for Raft [pingcap/raft#249](https://github.com/pingcap/raft-rs/issues/136).
+* [@nrc] opened a PR to enable both prost and rust-protobuf to be used in TiKV [#5379](https://github.com/tikv/tikv/pull/5379).
+* [@sticnarf] opened a series of PRs to begin enabling transaction support in the Rust client [tikv/client-rust#108](https://github.com/tikv/client-rust/pull/108), [tikv/client-rust#97](https://github.com/tikv/client-rust/pull/97), [tikv/client-rust#92](https://github.com/tikv/client-rust/pull/92).
+
 ## Notable issues
 
-## New contributors
+* In TiKV 2.1.15 and 2.1.16, region statistics may not be correct [#5306](https://github.com/tikv/tikv/issues/5306). Fixed in [#5415](https://github.com/tikv/tikv/pull/5415) and released as part of 2.1.17.
+* In TiKV 3.0 rc2 to 3.0.2, ReadIndex might fail to respond to requests due to a duplicate context [#4764](https://github.com/tikv/tikv/issues/4764). Fixed in [#5213](https://github.com/tikv/tikv/pull/5213) and released as part of 3.0.3.
+* In TiKV [#5291](https://github.com/tikv/tikv/pull/5291) we discovered a possible panic during region merge. Fixed in [#5291](https://github.com/tikv/tikv/pull/5291) and released in 3.0.3.
+* In [#4560](https://github.com/tikv/tikv/issues/4560) and [#4581](https://github.com/tikv/tikv/issues/4581) we discovered a possible panic while catching up after a restart. Fixed in [#4595](https://github.com/tikv/tikv/pull/4595) and released in 3.0.2.
 
+## Current projects
 
-We'd like to welcome the following new contributors to TiKV and thank them for their work!
+Here's some of the things our contributors have been working on over the last month:
+
+* [@sticnarf] and [@nrc] have been working on transaction support in our Rust Client.
+* [@fullstop000] has been working to implement follower replication in Raft, for future use in TiKV.
+* [@hicqu]has been leading the effort on supporting Joint Consensus in TiKV.
+* Several members of the [@pingcap] team have been working on further shrinking and optimizing TiKV's build graph, resulting in faster compile times and less dependencies.
+* [@zhouqiang-cl] and [@hoverbear] have been working on improving our release process and docker containers.
+* [@dcalvin] and [@hoverbear] have been overhauling the TiKV.org documentation to better support our users.
+* [@busyjay] and [@nrc] have been collaborating to land the final PRs to support using [Prost](https://github.com/danburkert/prost) library.
+* [@nrc] has been working on Prost to help land some optimizations.
+* [@overvenus] has been driving forward our new backup and restore process for TiKV.
+* [@MyonKeminta] has been leading on refining and hardening our experimental pessimistic transaction support.
```

Comment:
I think it's @youjiali1995 who leads pessimistic transaction.

Reply:
Oh! Thank you!